tensorflow_serving/batching/BUILD

```diff
@@ -114,3 +121,28 @@ cc_test(
         "@org_tensorflow//tensorflow/core:test",
     ],
 )
+
+cc_library(
+    name = "batching_util",
+    srcs = ["batching_util.cc"],
+    hdrs = ["batching_util.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@org_tensorflow//tensorflow/core:framework",
+        "@org_tensorflow//tensorflow/core:lib",
+        "@org_tensorflow//tensorflow/contrib/batching:basic_batch_scheduler",
```

Comment:
I haven't looked inside yet but it doesn't seem right that batching-util should depend on basic-batch-scheduler. What's the reason?